src/commands/protocols/get/sender/v1.0.0/v1-0-0-get-request-command.js

```diff
@@ -29,7 +30,7 @@ class GetRequestCommand extends ProtocolRequestCommand {
     async handleAck(command, responseData) {
         if (responseData?.nquads) {
             try {
-                await this.operationService.validateAssertion(
+                await this.validationService.validateAssertion(
```

Comment:
Why don't we use validation module for this? Why it's in service class?

Reply:
Because the function uses blockchain module manager